### DISCUSSION TITLE
Has something happened to Aspect Fit with an Image?

### DISCUSSION THREAD

**Discussion by munkii** [dotnet/maui#22903] [Q&A]
MAUI Version 8.0.40
Before I raise this as an Issue I just thought I'd check as it seems to obvious an issue to not be well known.

Previously in Xamarin Forms we used <Image/> to display local images in a Grid or StackLayout and they, by default were Centered. Now it seems like Image Aspect is always Fill or similar and Images cannot not be made to be Centered. As an example

With just the markup of  `<Image Margin="0,15" Source="{Binding Image}" />` in a StackLayout (Vertical)

### It looked like this.

<img src="https://github.com/dotnet/maui/assets/827342/cf96f510-bea3-45d4-9892-73333fa600fc" width="350">

### Now it looks like this

<img src="https://github.com/dotnet/maui/assets/827342/f3128ed4-6ddf-4513-b90d-7e981aa321b7" width="350">

**Comment by gautambjain** [comment]
Can you check if Aspect="Center" works as discussed here: https://github.com/dotnet/maui/issues/13101 . This issue is regarding padding and imagebutton, but looks similar to yours.

**Reply by munkii** [reply]
I've tried every possible value for Aspect and none of them get us back to where we were. FWIW This issue is Android and iOS

**Reply by gautambjain** [reply]
How about HorizontalOptions="Center" or "CenterAndExpand"?

**Reply by munkii** [reply]
Sadly not. I've tried all the HorizontalOptions (in iOS). I could hardcoded a HeightRequest but I don't want to rest to absolute numbers
